wordpress-mobile/WordPress-Android
# Remove GlobalStyleSupportFeatureConfig

This PR partially fixes #21412 by removing `GlobalStyleSupportFeatureConfig` from "Features in development." This debug setting defaulted to `true` so it wasn't necessary to keep it.

**Note:** [The issue](https://github.com/wordpress-mobile/WordPress-Android/issues/21412) mentions that we should also remove `SiteIntentQuestionFeatureConfig`, but I decided to do that in a separate PR.

## To Test:

* Visit the "Features in development" page of debug settings and verify that `GlobalStyleSupportFeatureConfig` is no longer there.
* Verify that the editor works as expected for both posts & pages
